services/galley/test/integration/API/Teams/Feature.hs

```diff
@@ -264,47 +246,42 @@ testSearchVisibility = do
         g <- view tsGalley
         Util.putTeamSearchVisibilityAvailableInternal g teamid val
 
-  assertFlagForbidden $ Util.getTeamFeatureFlag @Public.SearchVisibilityAvailableConfig nonMember tid
+  (owner, tid, member : _) <- Util.createBindingTeamWithNMembers 1
```

Comment:
```suggestion
  (owner, tid, [member]) <- Util.createBindingTeamWithNMembers 1
```

this is a super-nit-pick, but in tests i prefer incomplete patterns over over-general patterns that silently fail if something doesn't work as expected.